src/main/kotlin/com/dprint/formatter/DprintFormattingTask.kt

```diff
@@ -28,11 +27,7 @@ class DprintFormattingTask(
 ) {
     private var formattingIds = mutableListOf<Int>()
     private var isCancelled = false
-
-    /**
-     * Used when we want to cancel a format, so that we can cancel every future in the chain.
-     */
-    private val allFormatFutures = mutableListOf<CompletableFuture<FormatResult>>()
+    private var formatJob: Job? = null
```

Comment:
This only gets used in the cancel method and never actually gets a job assigned to it.